lua/warden/warden.lua

```diff
@@ -240,6 +243,46 @@ if SERVER then
 		end
 	end
 
+	function Warden.FreezeEntities(steamid)
+		local tbl = Warden.Players[steamid]
+		if tbl then
+			for entIndex, _ in pairs(tbl) do
+				local ent = Entity(entIndex)
+				for i = 0, ent:GetPhysicsObjectCount() - 1 do
+					local phys = ent:GetPhysicsObjectNum(i)
+					phys:EnableMotion(false)
+				end
+			end
+		end
+		hook.Run("WardenFreeze", steamid)
+	end
+
+	function Warden.CleanupEntities(steamid)
+		local tbl = Warden.Players[steamid]
+		if tbl then
+			for entIndex, _ in pairs(tbl) do
+				Entity(entIndex):Remove()
+			end
+		end
+		hook.Run("WardenCleanup", steamid)
+	end
+
+	function Warden.FreezeDisconnected()
+		for steamid, _ in pairs(Warden.Players) do
+			if Warden.PlayerIsDisconnected(steamid) then
+				Warden.FreezeEntities(steamid)
+			end
+		end
+	end
+
+	function Warden.CleanupDisconnected()
+		for steamid, _ in pairs(Warden.Players) do
+			if Warden.PlayerIsDisconnected(steamid) then
+				Warden.CleanupEntities(steamid)
+			end
+		end
+	end
```

Comment:
The two functions above here don't seem to be used anywhere, what is the purpose of them?

Reply:
Actually I guess there's other unused functions too so it doesn't really matter.